com.unity.netcode.adapter.utp/Tests/Editor/BatchedReceiveQueueTests.cs

```diff
@@ -72,5 +72,122 @@ public void BatchedReceiveQueue_MultipleMessages()
             Assert.AreEqual(default(ArraySegment<byte>), q.PopMessage());
             Assert.True(q.IsEmpty);
         }
+
+        [Test]
+        public void BatchedReceiveQueue_PartialMessage()
+        {
+            var dataLength = sizeof(int);
+
+            var data = new NativeArray<byte>(dataLength, Allocator.Temp);
+
+            var writer = new DataStreamWriter(data);
+            writer.WriteInt(42);
```

Comment:
What is the meaning of this magic number? A comment would be useful... a named const would be better.

Reply:
oh, nevermind, just realized this is a test... no big deal then.